elastio-lambda-stream-backup/readme.md

```diff
@@ -0,0 +1,68 @@
+Before starting, please check do you have aws-cliv2, and docker cli installed.
+
+1. From AWS console, select IAM, select Roles then create IAM role with "AdministratorAccess".  
+
+2. From the AWS console, select ECR and create an ECR repository for storing your docker image.
+
+3. Unpack archive and go into unpacked folder.
+# if you want use arm64/amd64 architecture you should use the instance in the same architecture to building image
```

Comment:
Why is this `#` here?  That will produce a Heading 1 heading, which isn't appropriate for something that is just a note.